public/animations/loader.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * The command is tied to script-imported library in DOM,
+ * and therefore cannot have a bodymovin variable defined
+ */
+// eslint-disable-next-line
+bodymovin.loadAnimation({
+  container: document.getElementById('bm'),
+  renderer: 'svg',
+  loop: true,
+  autoplay: true,
+  path: 'animations/data.json'
```

Comment:
Is this a series of multiple svg's that are stitched together to create an animation?